spec/views/goals/create.html.erb_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
Generated view specs can be safely deleted. Testing the views is normally done in the feature specs.